README.md

```diff
@@ -117,10 +117,12 @@ shows an example of a custom fetch service with proper authentication handling:
 import Service, { inject as service } from "@ember/service";
 import { handleUnauthorized } from "ember-simple-auth-oidc";
 import fetch from "fetch";
+import { waitFor } from "@ember/test-waiters";
 
 export default class FetchService extends Service {
   @service session;
 
+  @waitFor
```

Comment:
Using `@waitFor` is a decent suggestion since it does the main thing `ember-fetch` was helping with (installs a test waiter)